libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -750,8 +756,8 @@ - (NSString *)approvalURLForEndpoint:(NSString *)authorizeEndpoint
 
         if (!codeChallenge) {
             // Code verifier challenge:
-            //   - self.codeVerifier is a base64url-encoded random data string
-            //   - The code challenge sent here is an SHA-256 hash of self.codeVerifier, also base64url-encoded
+            //   - self.codeVerifier is a Base64 URL-Safe encoded (Note, not URL encoded) random data string
```

Comment:
I updated this comment since it is quite a significant difference.  These values are specifically being Base64 URL/Filename Safe Encoded, which is actually what the iOS code already does.  This caused some confusion when I was comparing it with the APEX code where many users are erroneously posting that the code challenge should be URL encoded when it actually must be Base 64 then Base 64 URL-safe encoded (which isn't the same).  I hope this being more specific will be less misleading in the future.